### DISCUSSION TITLE
hotbar mechanic

### DISCUSSION THREAD

**Discussion by wormylmao** [HaxeFlixel/flixel#2398] [Q&A]
for a small action game i want to make

**Comment by Gama11** [comment]
answer for a q&a discussion that somebody created

**Reply by wormylmao** [reply]
i dont see it

**Comment by RichardBray** [comment]
@wormylmao I'm not sure I understand your question. So you want to know how to implement a hotbar mechanic for a small game you want to make?
First of all, what is a hotbar mechanic?
